<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransWidget</class>
 <widget class="QWidget" name="TransWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Scale</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="ScaleX">
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="ScaleY">
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="ScaleZ">
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>250</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Rotation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="RotX">
      <property name="minimum">
       <double>-999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="RotY">
      <property name="minimum">
       <double>-999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="RotZ">
      <property name="minimum">
       <double>-999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Transform</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>246</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Position</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="PosX">
      <property name="minimum">
       <double>-999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="PosY">
      <property name="minimum">
       <double>-999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="PosZ">
      <property name="minimum">
       <double>-999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
